nautobot/extras/templates/extras/inc/graphqlquery_execute.html

```diff
@@ -0,0 +1,71 @@
+{% load helpers %}
+
+ <div class="panel-default">
+            <div class="panel-heading">
+                <strong>Response</strong>
+                <button class="btn btn-primary btn-xs pull-right" onclick="test_query()">Execute</button>
+            </div>
+            {% if object.variables %}
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Variables:</td>
+                        <td><textarea id="query_variables" class="form-control" style="height:86px">{{ object.variables|render_json:False }}</textarea></td>

```

Comment:
Where's this magic `86px` number come from? Why's it needed on this specific `textarea` but not on others?

Reply:
Sorry this was one of my debugging changes. It should work fine without this now.
```suggestion
                        <td><textarea id="query_variables" class="form-control">{{ object.variables|render_json:False }}</textarea></td>

```

Reply:
Updated as requested, should be resolved, let me If I miss anything